### PR TITLE
REVIEW | Add Services page

### PR DESCRIPTION
**Pull request by ayakakojimaak**
### PC
https://github.com/user-attachments/assets/4f549771-e8c1-4d01-bc09-1cf7ff914af7

### SP
https://github.com/user-attachments/assets/25aaf45d-26ff-4e59-bcbc-5bd0a2614bb8

### CODE REVIEW

File: src/pages/services.html [hunk 1]
```diff
@@ -1,126 +1,206 @@
...
-        </a>
-      </div>
-    </div>
-    <div id="js-nav-pc" class="header-nav d-lg-block">
-      <nav class="navbar navbar-expand-lg py-0">
-        <div class="container flex-column flex-lg-row position-relative">
-          <button id="js-nav-sp-btn-close" class="header-nav__close d-block d-lg-none">
-            <img src="public/assets/images/icon-cancel.svg" alt="close" width="40" />
+  <body>
+    <header class="header">
```
Comment (ayakakojimaak) [inline src/pages/services.html:11]: I don't modify the header or footer.

File: src/scss/partials/_common.scss [hunk 2]
```diff
@@ -1 +1,36 @@
 //common.scss
+
```
Comment (ayakakojimaak) [inline src/scss/partials/_common.scss:2]: This CSS is also used for the About page.

File: src/scss/partials/_variables.scss [hunk 3]
```diff
@@ -20,6 +20,16 @@ $complementary-brown: #a06b67;
 $font-title: "Baskervville", sans-serif;
 $font-text: "Lato", sans-serif;
 
+//TODO move to style.scss
```
Comment (ayakakojimaak) [inline src/scss/partials/_variables.scss:23]: I include this here to prevent any potential conflict.

**Review by kazuki-0418 — CHANGES_REQUESTED** [review]
@ayakakojimaak 
I'd like you to modify part of `content-headline__subtitle`
sentences are different from Figma design

image


## Figma design
image

**Comment by ayakakojimaak** [comment]
@kazuki-0418 
Thank you for your comment!
I modified sentences🙏
[fix text](https://github.com/Cornerstone-CICCC/web-1-final-project-term4/pull/3/commits/899e587dba1bbb69ada3afcb33699ab025cd91af)

Screenshot 2024-11-16 at 9 46 08 PM

**Review by kazuki-0418 — APPROVED** [review]
Thank you for completing the Service page!!👍

LGTM

**Pull request merged**